src/Process/3dTilesProcessing.js

```diff
@@ -120,21 +117,13 @@ function computeNodeSSE(camera, node) {
 }
 
 export function init3dTilesLayer(context, layer) {
-    if (layer.initialised) {
-        return;
-    }
-    layer.initialised = true;
-
-    Fetcher.json(layer.url, layer.networkOptions).then((tileset) => {
-        const urlPrefix = layer.url.slice(0, layer.url.lastIndexOf('/') + 1);
-        layer.tileIndex = new $3dTilesIndex(tileset, urlPrefix);
-        layer.asset = tileset.asset;
-        requestNewTile(context.view, context.scheduler, layer, tileset.root, undefined).then(
-            (tile) => {
-                layer.object3d.add(tile);
-                tile.updateMatrixWorld();
-                layer.root = tile;
-            });
+    layer.preprocessedPromise.then((layer) => {
```

Comment:
> Imho, not necessarily. We just have to make sure preupdate gives no element back in this case.

> if preupdate returns an empty list, it's not necessary. I like the fact preupdate is guaranteed to be called at each loop. WDYT?

It seems really strange that you would do some kind of processing on a layer that has not been completely defined yet.